modules/psap-special-resource-operator-using-manifests.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="deploy-simple-kmod-using-local-chart_{context}"]
+= Building and running the simple-kmod SpecialResource by using the local manifest for the SRO image
+
+The SRO image contains a local repository of Helm charts including the templates for deploying the simple-kmod kernel module. This example shows how to use the `SpecialResource` manifest to deploy the simple-kmod module.
```

Comment:
I think the meaning slightly changed in these edits, how about these changes:

```suggestion
[id="deploy-simple-kmod-using-local-chart_{context}"]
= Building and running the simple-kmod SpecialResource by using the templates from the SRO image

The SRO image contains a local repository of Helm charts including the templates for deploying the simple-kmod kernel module. In this example, the simple-kmod kernel module is used to show how SRO can manage a driver container which is defined in the internal SRO repository.
```

Reply:
fixed